ipynb/android-wloads/Android_YoutTube_Tests.ipynb

```diff
@@ -0,0 +1,500 @@
+      "2016-03-24 13:47:59,110 INFO    : Set plots time range to (0.000000, 45.829806)[s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Run the benchmark in all the configured governors\n",
+    "for governor in confs:\n",
+    "    test_dir = os.path.join(te.res_dir, governor)\n",
+    "    res = experiment(governor, test_dir)\n",
+    "    results[governor] = copy.deepcopy(res)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "# UI Performance Analysis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Frame Statistics for PERFORMANCE governor\n",
+      "Stats since: 8143079826214ns\n",
+      "Total frames rendered: 277\n",
+      "Janky frames: 22 (7.94%)\n",
+      "90th percentile: 12ms\n",
+      "95th percentile: 18ms\n",
+      "99th percentile: 73ms\n",
+      "\n",
+      "Frame Statistics for SCHED governor\n",
+      "Stats since: 8207240994471ns\n",
+      "Total frames rendered: 209\n",
+      "Janky frames: 24 (11.48%)\n",
+      "90th percentile: 16ms\n",
+      "95th percentile: 24ms\n",
+      "99th percentile: 53ms\n",
+      "\n",
+      "Frame Statistics for POWERSAVE governor\n",
+      "Stats since: 8278908871683ns\n",
+      "Total frames rendered: 257\n",
+      "Janky frames: 68 (26.46%)\n",
+      "90th percentile: 44ms\n",
+      "95th percentile: 61ms\n",
+      "99th percentile: 117ms\n",
+      "\n",
+      "Frame Statistics for ONDEMAND governor\n",
+      "Stats since: 8359985010454ns\n",
+      "Total frames rendered: 248\n",
+      "Janky frames: 30 (12.10%)\n",
+      "90th percentile: 16ms\n",
+      "95th percentile: 21ms\n",
+      "99th percentile: 57ms\n",
+      "\n",
+      "Frame Statistics for INTERACTIVE governor\n",
+      "Stats since: 8429068246521ns\n",
+      "Total frames rendered: 246\n",
+      "Janky frames: 40 (16.26%)\n",
+      "90th percentile: 19ms\n",
+      "95th percentile: 27ms\n",
+      "99th percentile: 57ms\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "for governor in confs:\n",
+    "    framestats_file = results[governor]['framestats_file']\n",
+    "    print \"Frame Statistics for {} governor\".format(governor.upper())\n",
+    "    !sed '/Stats since/,/99th/!d;/99th/q' $framestats_file\n",
+    "    print \"\""
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 2",
+   "language": "python",
+   "name": "python2"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 2
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython2",
+   "version": "2.7.6"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 0
+}
```

Comment:
Something like that:
```python
trace_file = results['interactive']['trace']
!xdg-open {trace_file}
```
can be used to open the collected SysTrace on a new browser window...